android/app/src/main/res/layout/launch_screen.xml

```diff
@@ -1,14 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/splash_bg"
-    android:gravity="center">
-    <ImageView
-        android:layout_width="200dp"
-        android:layout_height="113dp"
-        android:layout_marginTop="24dp"
-        android:src="@mipmap/icon"
-    />
-</LinearLayout>
\ No newline at end of file
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:layout_gravity="center">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="436dp"
+            android:gravity="top|center_horizontal">
+
+            <ImageView
+                android:id="@+id/Splash"
+                android:layout_width="172dp"
+                android:layout_height="236dp"
+                android:contentDescription="Discover, Plan, Love"
+                android:visibility="visible"
+                android:src="@mipmap/splash" />
+        </RelativeLayout>
+    </LinearLayout>
```

Comment:
Android studio is telling me I don't need to nest this much, but I don't see an easier way of achieving the layout I need to achieve. Any tips welcome.